src/github/revertPRViewProvider.ts

```diff
@@ -50,6 +48,33 @@ export class RevertPullRequestViewProvider extends BaseCreatePullRequestViewProv
 		return undefined;
 	}
 
+	protected async getCreateParams(): Promise<CreateParamsNew> {
+		const params = await super.getCreateParams();
+		params.canModifyBranches = false;
+		params.actionDetail = vscode.l10n.t('Reverting');
+		params.associatedExistingPullRequest = this.pullRequest.number;
+		return params;
+	}
+
+	private openAssociatedPullRequest() {
+		return openDescription(this.telemetry, this.pullRequest, undefined, this._folderRepositoryManager, false, true);
+	}
+
+	protected async _onDidReceiveMessage(message: IRequestMessage<any>) {
+		const result = await super._onDidReceiveMessage(message);
+		if (result !== this.MESSAGE_UNHANDLED) {
+			return;
+		}
+
+		switch (message.command) {
+			case 'pr.openAssociatedPullRequest':
+				return this.openAssociatedPullRequest();
+			default:
+				// Log error
+				vscode.window.showErrorMessage('Unsupported webview message');
+		}
```

Comment:
I think we added a lint rule to require break/return to the vscode repo recently, could help here to prevent accidents

Reply:
It does sound like a good one, but I haven't been able this rule.